project-management/src/main/java/life/qbic/projectmanagement/application/ProjectOverview.java

```diff
@@ -63,26 +62,6 @@ protected ProjectOverview() {
 
   }
 
-  public ProjectOverview(ProjectId id, String projectTitle, String projectCode,
-      Instant lastModified, String principalInvestigatorName, String projectManagerName,
-      String projectResponsibleName,
-      String ngsMeasurementCount, String pxpMeasurementCount,
-      Collection<String> usernames) {
-    this.id = Objects.requireNonNull(id, "project Id cannot be null");
-    this.projectTitle = Objects.requireNonNull(projectTitle, "projectTitle cannot be null");
-    this.projectCode = Objects.requireNonNull(projectCode, "project code cannot be null");
-    this.lastModified = Objects.requireNonNull(lastModified, "lastModified date cannot be null");
-    this.principalInvestigatorName = Objects.requireNonNull(principalInvestigatorName,
-        "principal investigator name cannot be null");
-    this.projectManagerName = Objects.requireNonNull(projectManagerName,
-        "project manager name cannot be null");
-    this.projectResponsibleName = projectResponsibleName;
-    this.ngsMeasurementCount = ngsMeasurementCount;
-    this.pxpMeasurementCount = pxpMeasurementCount;
-    this.collaboratorUserNames = Objects.requireNonNull(usernames,
-        "The collaborator user names cannot be null");
-  }
```

Comment:
I don't understand why this constructor was removed? Now the projectOverview only has a protected constructor? How are the fields initialized then 🤔

Reply:
There is no way to construct it as it is only a view. Further the constructor was unused.